app/models/booking.rb

```diff
@@ -1,4 +1,8 @@
 class Booking < ApplicationRecord
   belongs_to :pet
   belongs_to :user
+
+  validates :date, presence: true
+  validates :message, presence: true
+  validates :message, length: { minimum: 15 }
```

Comment:
set minimum to 6 characters and send message if character length < 6 like

validates :comment, length: { minimum: 6,
  too_short: “is too short (minimum is %{count} characters)” }